libfwupd/fwupd-enums.h

```diff
@@ -493,6 +493,15 @@ typedef enum {
  * Since: 1.7.1
  */
 #define FWUPD_DEVICE_FLAG_AFFECTS_FDE (1llu << 45)
+/**
+ *
+ * FWUPD_DEVICE_FLAG_NO_AUTO_REMOVE :
+ *
+ * The device is not auto removed.
+ * Since 1.7.3
+ */
+#define FWUPD_DEVICE_FLAG_NO_AUTO_REMOVE (1llu << 46)
+
```

Comment:
@hughsie should these be internal flags perhaps?  I'm thinking yes.
@krankun IOW define them in `libfwupdplugin/fu-device.h`

Reply:
Yes, I think internal too -- they're of no use to the end user.

Reply:
Oh but they're used in `src/fu-udev-backend.c`, so maybe not.

Reply:
done in https://github.com/fwupd/fwupd/pull/4046/commits/8d0d551cc1a62fbd54d246ec7a5ddb513bac95bd